### PR TITLE
Feature 5

### PR DESCRIPTION
**Pull request by MishaMazniak**
I added new folder "lesson_5" with file "index.html" (the table from lesson 5) and created "homeTask" it is folder with my homework

### CODE REVIEW

File: lesson_5/homeTask/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+              <th>Period</th>
+              <th>Position</th>
+            </tr>
+          </thead>
+          <tr>
+            <th align="left">04.2023 ...</th>
+            <td>
+              Operator CNC at
+              <a href="http://www.stemmann.com.pl/index.pl.php"
+                >Stemmann-Polska</a
```
Comment (RomanIlkiv) [inline lesson_5/homeTask/index.html:40]: bad formatting, should like these:
```
<a href="http://www.stemmann.com.pl/index.pl.php">
  Stemmann-Polska
</a>
```

File: lesson_5/homeTask/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>lesson_5 - HomeTask</title>
+  </head>
+  <body>
+    <div><h1>Mazniak Mykhailo</h1></div>
```
Comment (RomanIlkiv) [inline lesson_5/homeTask/index.html]: uncessary _div_ tag

File: lesson_5/homeTask/index.html (hunk 1 above)
Comment (MishaMazniak) [reply lesson_5/homeTask/index.html:40]: I know, but my Visual Studio (Prettier) does it past save

File: lesson_5/homeTask/index.html (hunk 2 above)
Comment (MishaMazniak) [reply lesson_5/homeTask/index.html]: ok

**Pull request merged**